src/modules/metadata/hooks.js

```diff
@@ -33,3 +33,16 @@ export const useOverviewSummary = () => {
   }, [dispatch, metadataService]);
   return useSelector((state) => state.overviewSummary);
 };
+
+export const useProjectJsonSchemaTypes = () => {
+  const dispatch = useDispatch();
+  const metadataService = useService("metadata");
+  useEffect(() => {
+    dispatch(fetchExtraPropertiesSchemaTypes());
+  }, [dispatch, metadataService]);
+  return useSelector((state) => ({
```

Comment:
selector shouldn't return an object, since it'll change every time. instead, there should be a deconstruction of `useProjects()` to the three properties, and then a returned memoized object of the three properties